content/en/database_monitoring/connect_dbm_and_apm.md

```diff
@@ -105,6 +105,8 @@ Datadog recommends setting the obfuscation mode to `obfuscate_and_normalize` for
   sql_obfuscation_mode: "obfuscate_and_normalize"
 ```
 
+*Warning:* Changing the obfuscation mode may alter the normalized SQL text. If you have monitors based on SQL text in APM traces, you may need to update them.
```

Comment:
Would it be better to put this is in a alert box? See the red box here: https://docs.datadoghq.com/agent/configuration/dual-shipping/?tab=helm

Reply:
If so, this would be the HTML for it:
```suggestion
<div class="alert alert-danger">Changing the obfuscation mode may alter the normalized SQL text. If you have monitors based on SQL text in APM traces, you may need to update them.</div>
```